src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemJsDisputedAnswerOverdueTask.java

```diff
@@ -59,7 +60,8 @@ public void run() {
             final BoolQueryBuilder query =
                 boolQuery()
                     .must(matchQuery(STATE, AwaitingAnswer))
-                    .must(matchQuery(String.format(DATA, IS_JUDICIAL_SEPARATION), YesOrNo.YES))
+                    .mustNot(matchQuery(String.format(DATA, SUPPLEMENTARY_CASE_TYPE), NA))
```

Comment:
Could these be a must(matchQuerry(String.format(DATA, SUPPLEMENTARY_CASE_TYPE), JUDICIAL_SEPARATION) ? Instead of checking for what it is not? similar to before the rework?

Reply:
We'd need to be able to match with an OR qualifier (it must match on js OR sep OR nullity) - I couldn't find any documentation or other examples when I looked before.  Do you know if this is possible?

Reply:
If it's not possible to match with OR, do we need to add an additional mostNot match for a null value?